<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Identity::Internal::IdentityWidget</class>
 <widget class="QWidget" name="Identity::Internal::IdentityWidget">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>525</width>
    <height>478</height>
   </rect>
  </property>
  <property name="sizePolicy">
   <sizepolicy hsizetype="Preferred" vsizetype="Minimum">
    <horstretch>0</horstretch>
    <verstretch>0</verstretch>
   </sizepolicy>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <layout class="QGridLayout" name="gridLayout_2">
   <item row="2" column="0">
    <widget class="QGroupBox" name="loginGroup">
     <property name="sizePolicy">
      <sizepolicy hsizetype="Preferred" vsizetype="Fixed">
       <horstretch>0</horstretch>
       <verstretch>0</verstretch>
      </sizepolicy>
     </property>
     <property name="title">
      <string>Login and password</string>
     </property>
     <layout class="QFormLayout" name="formLayout_3">
      <item row="0" column="0">
       <widget class="QLabel" name="loginLabel">
        <property name="text">
         <string>Login</string>
        </property>
       </widget>
      </item>
      <item row="1" column="0">
       <widget class="QLabel" name="passwordLabel">
        <property name="text">
         <string>Password</string>
        </property>
       </widget>
      </item>
      <item row="2" column="0">
       <widget class="QLabel" name="password2Label">
        <property name="text">
         <string>Confirm password</string>
        </property>
       </widget>
      </item>
      <item row="0" column="1">
       <widget class="Utils::LineEditEchoSwitcher" name="login"/>
      </item>
      <item row="1" column="1">
       <widget class="Utils::LineEditEchoSwitcher" name="password"/>
      </item>
      <item row="2" column="1">
       <widget class="Utils::LineEditEchoSwitcher" name="password2"/>
      </item>
     </layout>
    </widget>
   </item>
   <item row="1" column="0">
    <widget class="ZipCodes::ZipCodesWidget" name="zipcodesWidget" native="true"/>
   </item>
   <item row="0" column="0">
    <layout class="QGridLayout" name="gridLayout">
     <item row="3" column="1">
      <widget class="QLabel" name="secondNameLabel">
       <property name="sizePolicy">
        <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
         <horstretch>0</horstretch>
         <verstretch>0</verstretch>
        </sizepolicy>
       </property>
       <property name="minimumSize">
        <size>
         <width>0</width>
         <height>20</height>
        </size>
       </property>
       <property name="text">
        <string>Second name</string>
       </property>
       <property name="buddy">
        <cstring>secondName</cstring>
       </property>
      </widget>
     </item>
     <item row="0" column="0" rowspan="3">
      <widget class="Identity::Internal::ThemedGenderButton" name="photoButton">
       <property name="sizePolicy">
        <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
         <horstretch>0</horstretch>
         <verstretch>0</verstretch>
        </sizepolicy>
       </property>
       <property name="minimumSize">
        <size>
         <width>64</width>
         <height>64</height>
        </size>
       </property>
       <property name="maximumSize">
        <size>
         <width>64</width>
         <height>64</height>
        </size>
       </property>
       <property name="focusPolicy">
        <enum>Qt::ClickFocus</enum>
       </property>
       <property name="toolTip">
        <string>Click to add/change the photo</string>
       </property>
       <property name="text">
        <string/>
       </property>
       <property name="iconSize">
        <size>
         <width>50</width>
         <height>50</height>
        </size>
       </property>
       <property name="popupMode">
        <enum>QToolButton::InstantPopup</enum>
       </property>
      </widget>
     </item>
     <item row="5" column="1">
      <widget class="QLabel" name="genderLabel">
       <property name="sizePolicy">
        <sizepolicy hsizetype="Fixed" vsizetype="Preferred">
         <horstretch>0</horstretch>
         <verstretch>0</verstretch>
        </sizepolicy>
       </property>
       <property name="text">
        <string>Gender</string>
       </property>
      </widget>
     </item>
     <item row="6" column="2" colspan="2">
      <widget class="Utils::LanguageComboBox" name="language">
       <property name="minimumSize">
        <size>
         <width>0</width>
         <height>20</height>
        </size>
       </property>
      </widget>
     </item>
     <item row="2" column="1">
      <widget class="QLabel" name="firstnameLabel">
       <property name="sizePolicy">
        <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
         <horstretch>0</horstretch>
         <verstretch>0</verstretch>
        </sizepolicy>
       </property>
       <property name="minimumSize">
        <size>
         <width>0</width>
         <height>20</height>
        </size>
       </property>
       <property name="text">
        <string>First name</string>
       </property>
       <property name="buddy">
        <cstring>firstname</cstring>
       </property>
      </widget>
     </item>
     <item row="0" column="1" colspan="3">
      <widget class="QComboBox" name="titleCombo">
       <property name="sizePolicy">
        <sizepolicy hsizetype="Preferred" vsizetype="Fixed">
         <horstretch>0</horstretch>
         <verstretch>0</verstretch>
        </sizepolicy>
       </property>
       <property name="minimumSize">
        <size>
         <width>0</width>
         <height>20</height>
        </size>
       </property>
      </widget>
     </item>
     <item row="1" column="1" colspan="3">
      <widget class="QLineEdit" name="birthName">
       <property name="minimumSize">
        <size>
         <width>0</width>
         <height>20</height>
        </size>
       </property>
      </widget>
     </item>
     <item row="6" column="1">
      <widget class="QLabel" name="languageLabel">
       <property name="sizePolicy">
        <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
         <horstretch>0</horstretch>
         <verstretch>0</verstretch>
        </sizepolicy>
       </property>
       <property name="minimumSize">
        <size>
         <width>0</width>
         <height>20</height>
        </size>
       </property>
       <property name="text">
        <string>Language</string>
       </property>
      </widget>
     </item>
     <item row="4" column="1">
      <widget class="QLabel" name="dobLabel">
       <property name="sizePolicy">
        <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
         <horstretch>0</horstretch>
         <verstretch>0</verstretch>
        </sizepolicy>
       </property>
       <property name="minimumSize">
        <size>
         <width>0</width>
         <height>20</height>
        </size>
       </property>
       <property name="text">
        <string>Date of birth</string>
       </property>
       <property name="buddy">
        <cstring>dob</cstring>
       </property>
      </widget>
     </item>
     <item row="2" column="2" colspan="2">
      <widget class="QLineEdit" name="firstname">
       <property name="minimumSize">
        <size>
         <width>0</width>
         <height>20</height>
        </size>
       </property>
      </widget>
     </item>
     <item row="3" column="2" colspan="2">
      <widget class="QLineEdit" name="secondName">
       <property name="minimumSize">
        <size>
         <width>0</width>
         <height>20</height>
        </size>
       </property>
      </widget>
     </item>
     <item row="4" column="2" colspan="2">
      <widget class="Utils::BirthDayEdit" name="dob">
       <property name="minimumSize">
        <size>
         <width>0</width>
         <height>20</height>
        </size>
       </property>
      </widget>
     </item>
     <item row="5" column="2" colspan="2">
      <widget class="QComboBox" name="genderCombo">
       <property name="sizePolicy">
        <sizepolicy hsizetype="Preferred" vsizetype="Fixed">
         <horstretch>0</horstretch>
         <verstretch>0</verstretch>
        </sizepolicy>
       </property>
       <property name="minimumSize">
        <size>
         <width>0</width>
         <height>20</height>
        </size>
       </property>
      </widget>
     </item>
    </layout>
   </item>
  </layout>
 </widget>
 <customwidgets>
  <customwidget>
   <class>Utils::BirthDayEdit</class>
   <extends>QLineEdit</extends>
   <header location="global">utils/widgets/birthdayedit.h</header>
  </customwidget>
  <customwidget>
   <class>ZipCodes::ZipCodesWidget</class>
   <extends>QWidget</extends>
   <header location="global">zipcodesplugin/zipcodeswidget.h</header>
   <container>1</container>
  </customwidget>
  <customwidget>
   <class>Identity::Internal::ThemedGenderButton</class>
   <extends>QToolButton</extends>
   <header location="global">identityplugin/themedgenderbutton.h</header>
  </customwidget>
  <customwidget>
   <class>Utils::LanguageComboBox</class>
   <extends>QComboBox</extends>
   <header location="global">utils/widgets/languagecombobox.h</header>
  </customwidget>
  <customwidget>
   <class>Utils::LineEditEchoSwitcher</class>
   <extends>QLineEdit</extends>
   <header location="global">utils/widgets/lineeditechoswitcher.h</header>
  </customwidget>
 </customwidgets>
 <tabstops>
  <tabstop>titleCombo</tabstop>
  <tabstop>birthName</tabstop>
  <tabstop>firstname</tabstop>
  <tabstop>secondName</tabstop>
  <tabstop>dob</tabstop>
  <tabstop>genderCombo</tabstop>
  <tabstop>language</tabstop>
  <tabstop>login</tabstop>
  <tabstop>password</tabstop>
  <tabstop>password2</tabstop>
 </tabstops>
 <resources/>
 <connections/>
</ui>
